api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/EnrolleeExtService.java

```diff
@@ -79,34 +78,18 @@ public WithdrawnEnrollee withdrawEnrollee(AdminUser user, String enroleeShortcod
     if (!user.isSuperuser()) {
       throw new PermissionDeniedException("Not authoried to withdraw participants");
     }
-    Enrollee enrollee = authAdminUserToEnrollee(user, enroleeShortcode);
+    Enrollee enrollee = authUtilService.authAdminUserToEnrollee(user, enroleeShortcode);
     return withdrawnEnrolleeService.withdrawEnrollee(enrollee);
   }
 
   public KitRequest requestKit(AdminUser adminUser, String enrolleeShortcode, String kitTypeName)
       throws JsonProcessingException {
-    Enrollee enrollee = authAdminUserToEnrollee(adminUser, enrolleeShortcode);
+    Enrollee enrollee = authUtilService.authAdminUserToEnrollee(adminUser, enrolleeShortcode);
     return kitRequestService.requestKit(adminUser, enrollee, kitTypeName);
   }
 
   public Collection<KitRequest> getKitRequests(AdminUser adminUser, String enrolleeShortcode) {
-    Enrollee enrollee = authAdminUserToEnrollee(adminUser, enrolleeShortcode);
+    Enrollee enrollee = authUtilService.authAdminUserToEnrollee(adminUser, enrolleeShortcode);
     return kitRequestService.getKitRequests(adminUser, enrollee);
   }
```

Comment:
this enrolleeService is starting to get a little too kitchen-sink.   Pulling the auth method into main authUtils makes it easier to make other services related to enrollee operations